get_last_revision_titles.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+"""
+    get_last_revision.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Revisions` module: Get data including content of last revision of titles [[API]] and [[Main Page]]
+    MIT license
+"""
+import requests
+
+req_ssn = requests.Session()
+
+api_url = 'https://www.mediawiki.org/w/api.php'
+titles = 'API|Main Page'
```

Comment:
You could add some other fun page titles here. We are using almost same titles in all other examples too :-/

Reply:
One other thing to keep in mind here is that we keep the variable names same across this repository. For example a session request could be named `S`,  endpoint as `URL`, etc (see here for reference: https://github.com/srish/MediaWiki-Action-API-Code-Samples/blob/master/geosearch.py).